indra/sources/bel/api.py

```diff
@@ -133,6 +136,42 @@ def process_pybel_neighborhood(entity_names, network_type='graph_jsongz_url',
     return bp
 
 
+def process_text(bel: str, squeeze: bool = False):
+    """Process a single BEL statement and return the PybelProcessor
+    or a single statement if ``squeeze`` is True.
+
+    Parameters
+    ----------
+    bel : str
+        A BEL statement. See example below.
+
+    squeeze: bool
+        If squeeze and there's only one statement in the processor,
+        it will be unpacked.
+
+    Returns
+    -------
+    statements : Union[Statement, PybelProcessor]
+        A list of INDRA statments derived from the BEL statement.
+        If squeeze is true and there was only one statement, the
+        unpacked INDRA statement will be returned.
+
+    Examples
+    --------
+    >>> from indra.sources.bel import process_text
+    >>> bel_s = 'kin(p(FPLX:MEK)) -> kin(p(FPLX:ERK))'
+    >>> indra_s = process_text(bel_s, squeeze=True)
+    IncreaseAmount(MEK(), ERK())
```

Comment:
This part here is odd since we'd expect an Activation statement. I found that inside `process_text`, `pybel.parse` produces the expected intermediate result, i.e., 
```
{'source': {'modifier': 'Activity', 'function': 'Protein', 'concept': {'namespace': 'FPLX', 'name': 'MEK'},
 'effect': {'namespace': 'go', 'name': 'kinase activity', 'identifier': '0016301'}},
 'relation': 'directlyIncreases',
 'target': {'modifier': 'Activity', 'function': 'Protein', 'concept': {'namespace': 'FPLX', 'name': 'ERK'},
 'effect': {'namespace': 'go', 'name': 'kinase activity', 'identifier': '0016301'}}}
```
but then in `add_sbel_row`, the `effect` terms are ignored, so this is something that probably needs to be fixed in PyBEL.

Reply:
I was a bit hasty - the sbel JSON format isn't exactly the same as what comes out of `pybel.parse`. I added a bit of code to massage the JSON in 6d3f20497. Unfortunately this functionality isn't directly available in PyBEL - I think it would need a big restructure to make this elegant so I'd opt for the simple/fast solution